apps/api/src/donations/donations.controller.ts

```diff
@@ -67,6 +68,23 @@ export class DonationsController {
     return this.donationsService.create(createPaymentDto, user)
   }
 
+  @Post('create-bank-payment')
+  @Roles({
+    roles: [RealmViewSupporters.role, ViewSupporters.role],
+    mode: RoleMatchingMode.ANY,
+  })
+  createBankPayment(
+    @AuthenticatedUser()
+    user: KeycloakTokenParsed,
+    bankPaymentDto: CreateBankPaymentDto,
+  ) {
+    if (!user) {
+      throw new UnauthorizedException()
+    }
```

Comment:
Here the check for user seems redundant as the decorator `@Roles` makes sure we have an authorized user

However it's not going to fail even if we keep it that way